src/app/components/Button/Button.tsx

```diff
@@ -1,11 +1,21 @@
 import type { ReactNode } from 'react'
 import React from 'react'
+import styled from 'styled-components'
 
 type ButtonProps = {
   children: ReactNode
+  onClick: () => void
 }
-function Button({ children }: ButtonProps): JSX.Element {
-  return <button>{children}</button>
+function Button({ children, onClick }: ButtonProps): JSX.Element {
+  return <StyledButton onClick={onClick}>{children}</StyledButton>
 }
```

Comment:
There is no difference if you call `<Button onClick={() => console.log('Add concert')}>Add concerts</Button>` or `<StyledButton onClick={() => console.log('Add concert')}>Add concerts</StyledButton >` because they are nearly similar. 
You have a function, which forwards all properties to another function (function === component).

Thus, you can simplify this file. Actually, you only need this:

```tsx
import styled from 'styled-components'

const Button = styled.button`
  padding: 10px 20px;
  border-radius: 8px;
  background-color: var(--color-light-orange);
  color: var(--color-font-dark);
  border: 2px solid var(--color-blue);
`

export default Button
```